File: JS_SOLUTIONS/src/BIWEEKLY_CONTEST_124/Q1.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxOperations = function (nums) {
    let theSum = nums[0] + nums[1],
      ans = 0;
    for (let i = 0; i + 1 < nums.length; i += 2) {
      if (nums[i] + nums[i + 1] == theSum) {
        ans++;
      } else {
        break;
      }
    }
    return ans;
  };
  

File: JAVA_SOLUTIONS/src/BIWEEKLY_CONTEST_140/Q1.java
package BIWEEKLY_CONTEST_140;

public class Q1 {

    public static int minElement(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            int sum = makeSum(nums[i]);
            nums[i] = sum;
        }
        int min = nums[0];
        for (int i : nums) {
            min = Math.min(i, min);
        }
        return min;
    }

    public static int makeSum(int number) {
        int ans = 0;
        while (number > 0) {
            int lastDigit = number % 10;
            ans += lastDigit;
            number /= 10;
        }
        return ans;
    }

    public static void main(String[] args) {
        int nums[] = { 999,19,199 };
        int ans = minElement(nums);
        System.err.println(ans);
    }

}
